EIPS/eip-4626.md

```diff
@@ -28,20 +28,20 @@ A standard for tokenized Vaults will lower the integration effort for yield-bear
 
 ## Specification
 
-All ERC-4626 tokenized Vaults MUST implement ERC-20 to represent shares.
+All [EIP-4626](./eip-4626.md) tokenized Vaults MUST implement EIP-20 to represent shares.
 If a Vault is to be non-transferrable, it MAY revert on calls to `transfer` or `transferFrom`.
-The ERC-20 operations `balanceOf`, `transfer`, `totalSupply`, etc. operate on the Vault "shares"
+The EIP-20 operations `balanceOf`, `transfer`, `totalSupply`, etc. operate on the Vault "shares"
 which represent a claim to ownership on a fraction of the Vault's underlying holdings.
 
-All ERC-4626 tokenized Vaults MUST implement ERC-20's optional metadata extensions.
+All EIP-4626 tokenized Vaults MUST implement EIP-20's optional metadata extensions.
 The `name` and `symbol` functions SHOULD reflect the underlying token's `name` and `symbol` in some way.
 
-ERC-4626 tokenized Vaults MAY implement [EIP-2612](./eip-2612.md) to improve the UX of approving shares on various integrations.
+EIP-4626 tokenized Vaults MAY implement [EIP-2612](./eip-2612.md) to improve the UX of approving shares on various integrations.
```

Comment:
This is interesting... EIP-2612 isn't final yet. @MicahZoltu?

Reply:
Huh, I wonder why eipw didn't catch it cc @SamWilsn

Reply:
This was moved to final before eipw was added I think.

Reply:
I mean eipw running against this PR should be warning about EIP-2612 not being final, and I'm surprised it isn't (suggests a bug in eipw).

Reply:
Ah, it's showing an error now.